docs/projects/README.md

```diff
@@ -0,0 +1,469 @@
+work to begin if, for some reason, there is delay in creating issues for work at
+the end of a project. It also however makes sense to priorize issues that are
+"good first issues" and "help wanted" issues contributors outside of the core
+maintainers can help with.
+
+It is important to make sure dependencies are documented when creating these
+issues. Issues which depend on another issue or set of issues should be labeled
+as "blocked" and reference the blocking issue(s).
+
+Whenever possible, issues should be written so that an implementer can complete
+an atomic unit of work without needing to understand the full scope and
+technicalities of the project. The issue description should contain all
+necessary information to complete the issue. This can include linking to
+specific relevant sections of the existing implementation plan(s).
+
+### Implementation (`status: In Progress`)
+
+In the implementation phase, contributors begin work on the project. At this
+point the work has been divided into discrete tasks and ordered according to
+priority. Contributors should feel the benefits of all the planning and textual
+material generated in earlier phases. For the most part, contributors should
+feel as though they are just able to "do the work".
+
+It's likely during implementation we'll come across faulty assumptions or new
+problems which require design. It is important that any realization like this
+which blocks or delays the delivery of the project be clearly documented in the
+weekly updates.
+
+Ideally, issues stand on their own and do not require reading the full
+implementation plans. For some issues, complexity prevents this from being
+possible. If the contributor working on an issue is still unable to achieve
+clarity on the details of an issue, even after reading the relevant parts of the
+implementation plan, this would be a signal that details were not sufficiently
+captured during implementation.
+
+If at any point during implementation, it is discovered that the implementation
+planning did not sufficiently capture the details of the project, we will put
+the project back into a planning status. Someone familiar with the broader scope
+of the project will be assigned to explore the discovered discrepancies and
+sufficiently resolve them for the project to continue in confidence.
+
+This is distinct from when the implementation plan has explicitly called out
+that an ambiguity exists that will need to be resolved after a specific part of
+the implementation is complete. The main idea here is that if at any point
+during implementation, contributors are unclear on what they are meant to
+implement or how they're meant to implement it, and this wasn't an expected part
+of the project during planning, then something potentially big was missed during
+planning.
+
+In these cases, it is worth taking a step back to ensure that the full scope of
+the plan is defined and being followed.
```

Comment:
@sarayourfriend this entire section has be re-worked to integrate your (modified) suggestions about identifying when implementation planning is insufficient.

Reply:
Looks great, thanks.